docs/support/FAQ.md

```diff
@@ -200,20 +200,10 @@ See [“Introduction to SemVer” - Irina Gebauer](https://blog.greenkeeper.io/i
 
 ## Can I trust semantic-release with my releases?
 
-**semantic-release** has a full unit and integration test suite that tests `npm` publishes against the [npm-registry-couchapp](https://github.com/npm/npm-registry-couchapp).
+**semantic-release** has a full unit and integration test suite that tests `npm` publishes against the [verdaccio](https://www.npmjs.com/package/verdaccio).
```

Comment:
this isnt specific to this effort but is more accurate after [`995469f` (#2132)](https://github.com/semantic-release/semantic-release/pull/2132/commits/995469f7f621b4cde214cf902b9968422e45ae1e#diff-f4d7b7036e6b760c0003adac90ad4b043353c3591ccdd3c8547ab058bcb11708R8)